README.md

```diff
@@ -3,18 +3,13 @@
 This repository contains the Puppet modules for WSO2 Enterprise Integrator and the profiles related to Enterprise Integrator Analytics.
 
 ## Quick Start Guide
-1. Download an updated wso2ei-6.5.0.zip pack and copy it to the `<puppet_environment>/modules/common/files` directory in the **Puppetmaster**.
+1. Download an updated wso2ei-6.5.0.zip pack and copy it to the `<puppet_environment>/modules/ei_common/files/packs` directory in the **Puppetmaster**.
```

Comment:
Can we not mention that an updated pack is required here?

Reply:
Fixed https://github.com/wso2/puppet-ei/pull/62/commits/51e5dda4f212f0de1d63834c0b843906932b6044